app/default/application_routes.py

```diff
@@ -135,6 +137,46 @@ def decorator(*args, **kwargs):
     return decorator
 
 
+@application_bp.route("/fund-round/notification/<application_id>", methods=["GET", "POST"])
+@login_required
+@verify_application_owner_local
+def fund_round_close_notification(application_id):
+    application = get_application_data(application_id=application_id)
+    fund_data = get_fund_data(
+        fund_id=application.fund_id,
+        language=application.language,
+        as_dict=False,
+        ttl_hash=get_ttl_hash(Config.LRU_CACHE_TIME),
+    )
+    round_data = get_round_data_without_cache(
+        fund_id=application.fund_id, round_id=application.round_id, language=application.language
+    )
+    deadline = datetime.strptime(round_data.deadline, "%Y-%m-%dT%H:%M:%S")
+    # added this check sometimes if url forcefully change we should not show this notification unless if
+    # there is a round close
+    if datetime.now() > deadline:
```

Comment:
Small thing, but frontend already has a way to determine if a round is closed. Would be good to reuse what we have and also if the logic for closing a round ever changed, it's all in one place: https://github.com/communitiesuk/funding-service-design-frontend/blob/aff748a7e17f9390e7e99ea4f0424d2dd063168b/app/default/data.py#L337

Reply:
Fixed